### PR TITLE
Refactored code base, added some images that I made, add durability f…

### PR DESCRIPTION
**Pull request by DefecateRainbows**
…or upgrades, added repairing, and that's about it.

### CODE REVIEW

**Comment by StevenCopeland** [comment]
Hello, @DefecateRainbows! This is your first Pull Request that will be reviewed by [Ebert](https://ebertapp.io/), an automatic Code Review service. It will leave comments on this diff with potential issues and style violations found in the code as you push new commits. You can also see all the issues found on this Pull Request on its [review page](https://ebertapp.io/github/StevenCopeland/FILEPLAN/pulls/41). Please check our [documentation](https://docs.ebertapp.io) for more information.

File: script.js [hunk 1]
```diff
@@ -1,298 +1,284 @@
...
+		dataDrive.parent = addDataDrive;
+		dataDrive.initialize();
+	});
+	addDataDrive.text = "These'll temporarily boost clicking power.";
+	//addDataDrive.buysAvailable = 1;//Uncomment this to make this option go away after it's bought once.
+	addDataDrive.raiseCostBy = 0.0007;
+	addDataDrive.initialize();
+
+
+	//TODO: Take the organic stuff, I.E. the brains and brain banks, out of the dataBank upgrades, add to them(AI?), and make them a separate line of upgrades with a high starting price that increases much more gradually and a considerable boost to both click power and degree of generation.
```
Comment (StevenCopeland) [inline script.js]: TODO found

File: script.js [hunk 2]
```diff
@@ -1,298 +1,284 @@
...
+function ideaUpgrade(name, cost, functionToCallOnBuy){
+	upgrade.call(this, name, cost, '#ideaLister');
+
+	this.text = "Ooo, a fancy idea. I wonder if it's shiny. Buy it to find out if it's shiny, and unlock a new type of upgrade.";
+
+
+	this.onPurchase = function(){
+		functionToCallOnBuy();
+		this.kill();
+		this.initialize();
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 2 tabs.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 3]
```diff
@@ -1,298 +1,284 @@
...
+
+function ideaUpgrade(name, cost, functionToCallOnBuy){
+	upgrade.call(this, name, cost, '#ideaLister');
+
+	this.text = "Ooo, a fancy idea. I wonder if it's shiny. Buy it to find out if it's shiny, and unlock a new type of upgrade.";
+
+
+	this.onPurchase = function(){
+		functionToCallOnBuy();
+		this.kill();
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 2 tabs.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 4]
```diff
@@ -1,298 +1,284 @@
...
+
+
+function ideaUpgrade(name, cost, functionToCallOnBuy){
+	upgrade.call(this, name, cost, '#ideaLister');
+
+	this.text = "Ooo, a fancy idea. I wonder if it's shiny. Buy it to find out if it's shiny, and unlock a new type of upgrade.";
+
+
+	this.onPurchase = function(){
+		functionToCallOnBuy();
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 2 tabs.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 5]
```diff
@@ -1,298 +1,284 @@
...
-	}
+
+
+function ideaUpgrade(name, cost, functionToCallOnBuy){
+	upgrade.call(this, name, cost, '#ideaLister');
+
+	this.text = "Ooo, a fancy idea. I wonder if it's shiny. Buy it to find out if it's shiny, and unlock a new type of upgrade.";
+
+
+	this.onPurchase = function(){
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 1 tab.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 6]
```diff
@@ -1,298 +1,284 @@
...
-		clicker.perSec *= addition;
-		clicker.perClick *= addition;
-		clicker.bytes -= cost;
-	}
+
+
+function ideaUpgrade(name, cost, functionToCallOnBuy){
+	upgrade.call(this, name, cost, '#ideaLister');
+
+	this.text = "Ooo, a fancy idea. I wonder if it's shiny. Buy it to find out if it's shiny, and unlock a new type of upgrade.";
```
Comment (StevenCopeland) [inline script.js]: Strings must use singlequote.

[Read more about it here.](http://eslint.org/docs/rules/quotes)

Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 1 tab.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 7]
```diff
@@ -1,298 +1,284 @@
...
-var idea2 = function(name, cost, addition) {
-	if (clicker.bytes >= cost) {
-		clicker.perSec *= addition;
-		clicker.perClick *= addition;
-		clicker.bytes -= cost;
-	}
+
+
+function ideaUpgrade(name, cost, functionToCallOnBuy){
+	upgrade.call(this, name, cost, '#ideaLister');
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 1 tab.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 8]
```diff
@@ -1,298 +1,284 @@
...
+
+	var oldPurchase = this.onPurchase;//	These lines of code
+	this.onPurchase = function(){			//	just make it so that the code 
+		oldPurchase.call(this);					//	from the original upgrade is run, too.	
+
+		this.timesBought = this.timesBought%this.newTitles.length;//This makes the names and titles loop, so that your suffering really does never end... :D
+
+		this.increment = Math.round(this.increment+this.increment*(1/getBetterBy));
+		this.timeLapse = Math.round(this.timeLapse-this.timeLapse*(1-getBetterBy));
+	};
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 0 spaces but found 1 tab.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 9]
```diff
@@ -1,298 +1,284 @@
...
+
+	var oldPurchase = this.onPurchase;//	These lines of code
+	this.onPurchase = function(){			//	just make it so that the code 
+		oldPurchase.call(this);					//	from the original upgrade is run, too.	
+
+		this.timesBought = this.timesBought%this.newTitles.length;//This makes the names and titles loop, so that your suffering really does never end... :D
+
+		this.increment = Math.round(this.increment+this.increment*(1/getBetterBy));
+		this.timeLapse = Math.round(this.timeLapse-this.timeLapse*(1-getBetterBy));
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 2 tabs.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 10]
```diff
@@ -1,298 +1,284 @@
...
+		setTimeout(this.update.bind(this), this.timeLapse);
+	};
+
+	var oldPurchase = this.onPurchase;//	These lines of code
+	this.onPurchase = function(){			//	just make it so that the code 
+		oldPurchase.call(this);					//	from the original upgrade is run, too.	
+
+		this.timesBought = this.timesBought%this.newTitles.length;//This makes the names and titles loop, so that your suffering really does never end... :D
+
+		this.increment = Math.round(this.increment+this.increment*(1/getBetterBy));
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 2 tabs.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 11]
```diff
@@ -1,298 +1,284 @@
...
+		updateBytes();
+
+		setTimeout(this.update.bind(this), this.timeLapse);
+	};
+
+	var oldPurchase = this.onPurchase;//	These lines of code
+	this.onPurchase = function(){			//	just make it so that the code 
+		oldPurchase.call(this);					//	from the original upgrade is run, too.	
+
+		this.timesBought = this.timesBought%this.newTitles.length;//This makes the names and titles loop, so that your suffering really does never end... :D
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 2 tabs.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 12]
```diff
@@ -1,298 +1,284 @@
...
+	this.update = function(){
+		fileSystem.bytes = fileSystem.bytes + this.increment;
+		updateBytes();
+
+		setTimeout(this.update.bind(this), this.timeLapse);
+	};
+
+	var oldPurchase = this.onPurchase;//	These lines of code
+	this.onPurchase = function(){			//	just make it so that the code 
+		oldPurchase.call(this);					//	from the original upgrade is run, too.
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 2 tabs.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 13]
```diff
@@ -1,298 +1,284 @@
...
+
+	this.update = function(){
+		fileSystem.bytes = fileSystem.bytes + this.increment;
+		updateBytes();
+
+		setTimeout(this.update.bind(this), this.timeLapse);
+	};
+
+	var oldPurchase = this.onPurchase;//	These lines of code
+	this.onPurchase = function(){			//	just make it so that the code
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 1 tab.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 14]
```diff
@@ -1,298 +1,284 @@
...
+	this.timeLapse = timeLapse;//so we can change 'em later.
+
+	this.update = function(){
+		fileSystem.bytes = fileSystem.bytes + this.increment;
+		updateBytes();
+
+		setTimeout(this.update.bind(this), this.timeLapse);
+	};
+
+	var oldPurchase = this.onPurchase;//	These lines of code
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 1 tab.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 15]
```diff
@@ -1,298 +1,284 @@
...
+	this.text = "This upgrade increments your storage every once in a certain amount of time.";
+	this.increment = increment;//We'll bind these guys to our upgrade,
+	this.timeLapse = timeLapse;//so we can change 'em later.
+
+	this.update = function(){
+		fileSystem.bytes = fileSystem.bytes + this.increment;
+		updateBytes();
+
+		setTimeout(this.update.bind(this), this.timeLapse);
+	};
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 0 spaces but found 1 tab.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 16]
```diff
@@ -1,298 +1,284 @@
...
+
+	this.text = "This upgrade increments your storage every once in a certain amount of time.";
+	this.increment = increment;//We'll bind these guys to our upgrade,
+	this.timeLapse = timeLapse;//so we can change 'em later.
+
+	this.update = function(){
+		fileSystem.bytes = fileSystem.bytes + this.increment;
+		updateBytes();
+
+		setTimeout(this.update.bind(this), this.timeLapse);
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 2 tabs.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 17]
```diff
@@ -1,298 +1,284 @@
...
+function generationUpgrade(name, cost, getBetterBy, timeLapse, increment){
+	upgrade.call(this, name, cost, '#upgradeHolder');
+
+	this.text = "This upgrade increments your storage every once in a certain amount of time.";
+	this.increment = increment;//We'll bind these guys to our upgrade,
+	this.timeLapse = timeLapse;//so we can change 'em later.
+
+	this.update = function(){
+		fileSystem.bytes = fileSystem.bytes + this.increment;
+		updateBytes();
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 2 tabs.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 18]
```diff
@@ -1,298 +1,284 @@
...
+
+function generationUpgrade(name, cost, getBetterBy, timeLapse, increment){
+	upgrade.call(this, name, cost, '#upgradeHolder');
+
+	this.text = "This upgrade increments your storage every once in a certain amount of time.";
+	this.increment = increment;//We'll bind these guys to our upgrade,
+	this.timeLapse = timeLapse;//so we can change 'em later.
+
+	this.update = function(){
+		fileSystem.bytes = fileSystem.bytes + this.increment;
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 2 tabs.

[Read more about it here.](http://eslint.org/docs/rules/indent)

File: script.js [hunk 19]
```diff
@@ -1,298 +1,284 @@
...
+
+
+function generationUpgrade(name, cost, getBetterBy, timeLapse, increment){
+	upgrade.call(this, name, cost, '#upgradeHolder');
+
+	this.text = "This upgrade increments your storage every once in a certain amount of time.";
+	this.increment = increment;//We'll bind these guys to our upgrade,
+	this.timeLapse = timeLapse;//so we can change 'em later.
+
+	this.update = function(){
```
Comment (StevenCopeland) [inline script.js]: Expected indentation of 2 spaces but found 1 tab.

[Read more about it here.](http://eslint.org/docs/rules/indent)

**Comment by StevenCopeland** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 23 possible new issues (including those that may have been commented here).
* 11 fixed issues! 🎉

But beware that this branch is **[23 commits behind the `StevenCopeland:master` branch](https://github.com/StevenCopeland/FILEPLAN/compare/DefecateRainbows:master...StevenCopeland:master)**, and a review of an up to date branch would produce more accurate results.

You can see more details about this review at https://ebertapp.io/github/StevenCopeland/FILEPLAN/pulls/41.

File: script.js [hunk 20]
```diff
@@ -1,298 +1,284 @@
...
+		recursiveUpgrade.parent = addRecursiveUpgrade;
+		recursiveUpgrade.initialize();
+		recursiveUpgrade.update();
+	});
+	addRecursiveUpgrade.text = "These'll make storage space automatically!";
+	//addRecursiveUpgrade.buysAvailable = 1;//Uncomment this to make this option go away after it's bought once.
+	addRecursiveUpgrade.raiseCostBy = 0.1;
+	addRecursiveUpgrade.initialize();
+
+	repairAll.text = "Duct tape, anyone?";
```
[the remark this answers is not in the document]
Comment (DefecateRainbows) [reply script.js]: wrong

File: script.js [hunk 21]
```diff
@@ -1,298 +1,284 @@
...
+	addDataDrive.text = "These'll temporarily boost clicking power.";
+	//addDataDrive.buysAvailable = 1;//Uncomment this to make this option go away after it's bought once.
+	addDataDrive.raiseCostBy = 0.0007;
+	addDataDrive.initialize();
+
+
+	//TODO: Take the organic stuff, I.E. the brains and brain banks, out of the dataBank upgrades, add to them(AI?), and make them a separate line of upgrades with a high starting price that increases much more gradually and a considerable boost to both click power and degree of generation.
+	
+
+	var addRecursiveUpgrade = new ideaUpgrade('Add Circuit', 5000, function(){
```
[the remark this answers is not in the document]
Comment (DefecateRainbows) [reply script.js]: wrong
In JS that's common practice

File: script.js [hunk 22]
```diff
@@ -1,298 +1,284 @@
...
-	$("#byteUpgradeIdea2").hover(function() {
-		$("#upgrade10").show();
-	}, function() {
-		$("#upgrade10").hide();
-	});
+//makeTextCamelCase.
+function camelize(str){
+  return str.replace(/(?:^\w|[A-Z]|\b\w|\s+)/g, function(match, index) {
+    if (+match === 0) return ""; // or if (/\s+/.test(match)) for white spaces
+    return index == 0 ? match.toLowerCase() : match.toUpperCase();
```
[the remark this answers is not in the document]
Comment (DefecateRainbows) [reply script.js]: That's going to break everything.

File: script.js [hunk 23]
```diff
@@ -1,298 +1,284 @@
...
-setInterval(function() { showCode(80, "idea1", "one") },10);
-*/
+		document.getElementById("dateCount").innerHTML = ' ' + Math.round(new Date().getTime()/1000 - startDate);
+	}, 1000);
+
+	var addDataDrive = new ideaUpgrade('Add Data Drive', 10, function(){
+		var dataDrive = new clickerUpgrade("Magnetic Tape", 15, 0.5);
+		dataDrive.text = "Heavy and holy.";
+		dataDrive.newTitles = ['Compact Diskette', 		'Disk', 			 "USB", 					 'SD Card',													  			 "Hard Drive",     		 				"Web Server",                  "Human Brain",  "Human Brain Bank",    "Mutant Brain", "Mutant Brain Bank",    "Alien Brain", "Alien Brain Bank", 					 "Wheel of Cheese",		  "Fractal Engine",									 "Portal Gun",								 "Singularity"];
+		dataDrive.newTexts =  ['Woah. Fancy.', 				"It's shiny!", "Data crystals!", "So much space on so little surface area!", "Don't spill coffee on it!", "Dude! It has flashy lights!", "Huh. Sticky.", "Organic Technology!", "Still sticky!", "Synthetic-bio-tech!", "Not sticky!", "Extraterrestrial-Bio Tech!", "Who'd have known...",	"Best data storage known to man.", "Extra dimensional storage!", "Do not throw at bear."]
```
[the remark this answers is not in the document]
Comment (DefecateRainbows) [reply script.js]: Wow... thanks.

File: script.js [hunk 24]
```diff
@@ -1,298 +1,284 @@
...
+	addRecursiveUpgrade.text = "These'll make storage space automatically!";
+	//addRecursiveUpgrade.buysAvailable = 1;//Uncomment this to make this option go away after it's bought once.
+	addRecursiveUpgrade.raiseCostBy = 0.1;
+	addRecursiveUpgrade.initialize();
+
+	repairAll.text = "Duct tape, anyone?";
+	repairAll.initialize();
+});
+
+var repairAll = new ideaUpgrade('Repair All', 0, function(){
```
[the remark this answers is not in the document]
Comment (DefecateRainbows) [reply script.js]: YES IT SHOULD

File: script.js [hunk 25]
```diff
@@ -1,298 +1,284 @@
...
-};
-
-// Minified code functions
-var idea = function(name, cost, addition) {
-	if (clicker.bytes >= cost) {
-		clicker.perClick += addition;
-		clicker.bytes -= cost;
-	}
+	this.initialize = function(){
+		if($(whereTo).find('#'+this.id).length > 0)return false;
```
[the remark this answers is not in the document]
Comment (DefecateRainbows) [reply script.js]: That's, again, common practice in JS. Codacy? More like idiocy.

File: script.js [hunk 26]
```diff
@@ -1,298 +1,284 @@
...
-// IDEA: Add graphics.
-
-/*
-IDEA code:
-setInterval(function() { showCode(80, "idea1", "one") },10);
-*/
+		document.getElementById("dateCount").innerHTML = ' ' + Math.round(new Date().getTime()/1000 - startDate);
+	}, 1000);
+
+	var addDataDrive = new ideaUpgrade('Add Data Drive', 10, function(){
```
[the remark this answers is not in the document]
Comment (DefecateRainbows) [reply script.js]: UGH!

File: script.js [hunk 27]
```diff
@@ -1,298 +1,284 @@
...
+		var dataDrive = new clickerUpgrade("Magnetic Tape", 15, 0.5);
```
[the remark this answers is not in the document]
Comment (DefecateRainbows) [reply script.js]: WHY!?!? You are wrong!

File: script.js [hunk 28]
```diff
@@ -1,298 +1,284 @@
...
+
+				this.title = this.newTitles[this.timesBought] || this.title;//									and then we'lll change the name, if we can think of any new ones.
+				this.text = this.newTexts[this.timesBought] || this.text;//											Oh, and the description, too.
+
+				this.timesBought = this.timesBought + 1;//																			Then we'll record your purchase,
+
+				this.onPurchase();// 																														And then do any extra stuff that depends upon this individual upgrade.
+
+				this.upgradesTriggeredOnBuys.forEach(function(element){							
+					if(element.onHowManyBuys >= this.timesBought)element.upgrade.initialize();//	Also, we'll trigger any cascading upgrades, and then...
```
[the remark this answers is not in the document]
Comment (DefecateRainbows) [reply script.js]: Shut up.

**Comment by StevenCopeland** [comment]
@DefecateRainbows Sorry! This bot is really ridiculous!

File: script.js [hunk 29]
```diff
@@ -1,298 +1,284 @@
...
+	//addDataDrive.buysAvailable = 1;//Uncomment this to make this option go away after it's bought once.
+	addDataDrive.raiseCostBy = 0.0007;
+	addDataDrive.initialize();
+
+
+	//TODO: Take the organic stuff, I.E. the brains and brain banks, out of the dataBank upgrades, add to them(AI?), and make them a separate line of upgrades with a high starting price that increases much more gradually and a considerable boost to both click power and degree of generation.
+	
+
+	var addRecursiveUpgrade = new ideaUpgrade('Add Circuit', 5000, function(){
+		var recursiveUpgrade = new generationUpgrade('Complete Circuit', 5000, 0.7, 500, 55);
```
[the remark this answers is not in the document]
Comment (DefecateRainbows) [reply script.js]: String quote type is arbitrary.

File: script.js [hunk 30]
```diff
@@ -1,298 +1,284 @@
...
 
-	$("#byteUpgradeIdea2").hover(function() {
-		$("#upgrade10").show();
-	}, function() {
-		$("#upgrade10").hide();
-	});
+//makeTextCamelCase.
+function camelize(str){
+  return str.replace(/(?:^\w|[A-Z]|\b\w|\s+)/g, function(match, index) {
+    if (+match === 0) return ""; // or if (/\s+/.test(match)) for white spaces
```
[the remark this answers is not in the document]
Comment (DefecateRainbows) [reply script.js]: You can do that in JavaScript... everyone knows that....

**Comment by StevenCopeland** [comment]
Do you mind sending the github pages url? I'll be back on here in about 20 minutes.

**Comment by DefecateRainbows** [comment]
Umm... why do you use these bots? I'm sorry, but they're just not accurate. I mean, stuff like jshint.com is great for sticking in those semi-colon that you missed, but when programs try to edit code to an extent any greater than that, they get kinda screwy. Fortunately, computers can't program themselves, so they don't know what's best.

**Comment by StevenCopeland** [comment]
I agree with you. The bots are kind of annoying but when I used them, not many people contributed(none) so I just used them for myself.

File: script.js [hunk 31]
```diff
@@ -1,262 +1,281 @@
...
+	repairAll.initialize();
+
+	fileSystem.bytes = Math.round((fileSystem.bytes + fileSystem.clickStrength + totalBoost)*100)/100;
+	updateBytes();
+}
+
+//End of Game Logic.
+
+
+//END OF FILEPLAN. Have a nice day!
```
Comment (StevenCopeland) [inline script.js]: Parsing error: Unexpected token

File: style.css [hunk 32]
```diff
@@ -210,21 +246,26 @@ h2 {
...
 	float:left;
-	-webkit-margin-before: 3px;
-    -webkit-margin-after: 3px;
-    -webkit-margin-start: 0px;
-    -webkit-margin-end: 0px;
-    background-color:black;
+	-webkit-margin-before: 5px;
+    -webkit-margin-after: 0px;
+    -webkit-margin-start: 5px;
+    -webkit-margin-end: 7px;
```
Comment (StevenCopeland) [inline style.css]: The property -moz-margin-end is compatible with -webkit-margin-end and should be included as well.

File: style.css [hunk 33]
```diff
@@ -210,21 +246,26 @@ h2 {
...
 .byteUpgrades img {
 	float:left;
-	-webkit-margin-before: 3px;
-    -webkit-margin-after: 3px;
-    -webkit-margin-start: 0px;
-    -webkit-margin-end: 0px;
-    background-color:black;
+	-webkit-margin-before: 5px;
+    -webkit-margin-after: 0px;
+    -webkit-margin-start: 5px;
```
Comment (StevenCopeland) [inline style.css]: The property -moz-margin-start is compatible with -webkit-margin-start and should be included as well.

File: style.css [hunk 34]
```diff
@@ -210,21 +246,26 @@ h2 {
...
 
 .byteUpgrades img {
 	float:left;
-	-webkit-margin-before: 3px;
-    -webkit-margin-after: 3px;
-    -webkit-margin-start: 0px;
-    -webkit-margin-end: 0px;
-    background-color:black;
+	-webkit-margin-before: 5px;
+    -webkit-margin-after: 0px;
```
Comment (StevenCopeland) [inline style.css]: Values of 0 shouldn't have units specified.

File: style.css [hunk 35]
```diff
@@ -210,21 +246,26 @@ h2 {
 }
 
 .byteUpgrades {
+	position:relative;
 	border-bottom:1px solid white;
-	-webkit-margin-start: 20px;
+	-webkit-margin-start: 0px;
```
Comment (StevenCopeland) [inline style.css]: Values of 0 shouldn't have units specified.

Comment (StevenCopeland) [inline style.css]: The property -moz-margin-start is compatible with -webkit-margin-start and should be included as well.

File: style.css [hunk 36]
```diff
@@ -180,21 +213,24 @@ h2 {
...
-	-webkit-margin-start: 20px;
-    -webkit-margin-end: 20px;
+	-webkit-margin-start: 0px;
+    -webkit-margin-end: 0px;
     -webkit-margin-before: 5px;
     -webkit-margin-after: 5px
 }
 
-#byteUpgradeSec1 img {
+.upgradeDiv img {
```
Comment (StevenCopeland) [inline style.css]: Rule doesn't have all its properties in alphabetical ordered.

File: style.css [hunk 37]
```diff
@@ -180,21 +213,24 @@ h2 {
...
 }
 
-#byteUpgradeSec1 {
+.upgradeDiv {
 	border-top:1px solid white;
 	border-bottom:1px solid white;
-	-webkit-margin-start: 20px;
-    -webkit-margin-end: 20px;
+	-webkit-margin-start: 0px;
+    -webkit-margin-end: 0px;
```
Comment (StevenCopeland) [inline style.css]: Values of 0 shouldn't have units specified.

Comment (StevenCopeland) [inline style.css]: The property -moz-margin-end is compatible with -webkit-margin-end and should be included as well.

File: style.css [hunk 38]
```diff
@@ -180,21 +213,24 @@ h2 {
...
 	font-style:italic;
 }
 
-#byteUpgradeSec1 {
+.upgradeDiv {
 	border-top:1px solid white;
 	border-bottom:1px solid white;
-	-webkit-margin-start: 20px;
-    -webkit-margin-end: 20px;
+	-webkit-margin-start: 0px;
```
Comment (StevenCopeland) [inline style.css]: The property -moz-margin-start is compatible with -webkit-margin-start and should be included as well.

Comment (StevenCopeland) [inline style.css]: Values of 0 shouldn't have units specified.

File: style.css [hunk 39]
```diff
@@ -180,21 +213,24 @@ h2 {
...
+	position:absolute;
+	bottom:5px;
+	right:10px;
 	color:red;
 	font-size:11px;
 	font-style:italic;
 }
 
-#byteUpgradeSec1 {
+.upgradeDiv {
```
Comment (StevenCopeland) [inline style.css]: Rule doesn't have all its properties in alphabetical ordered.

File: style.css [hunk 40]
```diff
@@ -133,33 +150,49 @@ img {
...
+}
+
+#upgradeHolder {
+	float:center;
+	display: inline-block;
+    font-size: 0.67em;
+    width:100%;
+    -webkit-margin-before: 1em;
+    -webkit-margin-after: 1em;
+    -webkit-margin-start: 5px;
```
Comment (StevenCopeland) [inline style.css]: The property -moz-margin-start is compatible with -webkit-margin-start and should be included as well.

File: style.css [hunk 41]
```diff
@@ -133,33 +150,49 @@ img {
...
-    -webkit-margin-before: 0em;
+    -webkit-margin-before: 1em;
     -webkit-margin-after: 1em;
-    -webkit-margin-start: 0px;
+    -webkit-margin-start: 5px;
+    -webkit-margin-end: 5px;
+}
+
+#upgradeHolder {
+	float:center;
```
Comment (StevenCopeland) [inline style.css]: Expected (left | right | none) but found 'center'.

Comment (StevenCopeland) [inline style.css]: float can't be used with display: inline-block.

File: style.css [hunk 42]
```diff
@@ -133,33 +150,49 @@ img {
...
     font-size: 0.67em;
-    -webkit-margin-before: 0em;
+    -webkit-margin-before: 1em;
     -webkit-margin-after: 1em;
-    -webkit-margin-start: 0px;
+    -webkit-margin-start: 5px;
+    -webkit-margin-end: 5px;
+}
+
+#upgradeHolder {
```
Comment (StevenCopeland) [inline style.css]: Rule doesn't have all its properties in alphabetical ordered.

Comment (StevenCopeland) [inline style.css]: Don't use IDs in selectors.

File: style.css [hunk 43]
```diff
@@ -133,33 +150,49 @@ img {
...
+	font-weight:normal;
+	text-align:center;
+	display: inline-block;
     font-size: 0.67em;
-    -webkit-margin-before: 0em;
+    -webkit-margin-before: 1em;
     -webkit-margin-after: 1em;
-    -webkit-margin-start: 0px;
+    -webkit-margin-start: 5px;
+    -webkit-margin-end: 5px;
```
Comment (StevenCopeland) [inline style.css]: The property -moz-margin-end is compatible with -webkit-margin-end and should be included as well.

File: style.css [hunk 44]
```diff
@@ -133,33 +150,49 @@ img {
...
+#upgrades{
+	font-weight:normal;
+	text-align:center;
+	display: inline-block;
     font-size: 0.67em;
-    -webkit-margin-before: 0em;
+    -webkit-margin-before: 1em;
     -webkit-margin-after: 1em;
-    -webkit-margin-start: 0px;
+    -webkit-margin-start: 5px;
```
Comment (StevenCopeland) [inline style.css]: The property -moz-margin-start is compatible with -webkit-margin-start and should be included as well.

File: style.css [hunk 45]
```diff
@@ -133,33 +150,49 @@ img {
...
     -webkit-margin-end: 5px;
-    font-weight: bold;
 }
 
-#upgrades {
-	text-align:right;
-	display: block;
+#upgrades{
```
Comment (StevenCopeland) [inline style.css]: Rule doesn't have all its properties in alphabetical ordered.

**Comment by StevenCopeland** [comment]
Ebert has finished reviewing this Pull Request and has found:

* 7 possible new issues (including those that may have been commented here).
* 28 fixed issues! 🎉

But beware that this branch is **[1 commit behind the `StevenCopeland:master` branch](https://github.com/StevenCopeland/FILEPLAN/compare/DefecateRainbows:master...StevenCopeland:master)**, and a review of an up to date branch would produce more accurate results.

You can see more details about this review at https://ebertapp.io/github/StevenCopeland/FILEPLAN/pulls/41.

**Pull request merged**